prow/config/jobs/istio.yaml

```diff
@@ -221,6 +221,194 @@ jobs:
+    type: presubmit
+    command: [prow/integ-suite-local.sh, test.integration.security.local.presubmit]
+
+  - name: integ-framework-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.framework.kube.presubmit]
+    requirements: [kind]
+
+  - name: integ-istioctl-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.istioctl.kube.presubmit]
+    requirements: [kind]
+
+  - name: integ-galley-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.galley.kube.presubmit]
+    requirements: [kind]
+
+  - name: integ-mixer-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.mixer.kube.presubmit]
+    requirements: [kind]
+
+  - name: integ-pilot-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.pilot.kube.presubmit]
+    requirements: [kind]
+
+  - name: integ-security-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.security.kube.presubmit]
+    requirements: [kind]
+
+  - name: integ-telemetry-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.telemetry.kube.presubmit]
+    requirements: [kind]
+
+  - name: integ-new-install-k8s-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/integ-suite-kind.sh, test.integration.new.installer]
+    requirements: [kind]
+
+  - name: e2e-mixer-no_auth-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    command: [prow/e2e-kind-suite.sh, --single_test, e2e_mixer]
+    requirements: [kind]
+
+  - name: pilot-e2e-envoyv2-v1alpha3-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    command: [prow/e2e-kind-suite.sh, --single_test, e2e_pilotv2_v1alpha3]
+    requirements: [kind]
+
+  # TODO migrate to kind once we have metric server in kind
+  - name: e2e-dashboard-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    command: [prow/e2e-dashboard.sh]
+    requirements: [gcp]
```

Comment:
This one also won't work I think, due to the scripts calling `gcloud` to authenticate. We should try it though, I am not certain